vars/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+server_ip: 10.254.0.1
+service_ip: 10.254.0.143
+nb_service_ip: 10.254.0.145
+kube_config_dir: /etc/kubernetes
+kubectl_bin_dir: "{{ bin_dir | default('/usr/local/bin') }}"
```

Comment:
These things are configurable, so role vars is probably not the best place for them - the role should be portable between environments. You can use group_vars, host vars, inventory host vars, or extra vars (`-e @<file>`) for this.

This is quite useful reading: http://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable, and the layout tips here are useful (but not hard and fast rules): http://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html.

Reply:
that seems logical!